packages/clarity-js/src/interaction/scroll.ts

```diff
@@ -93,6 +100,15 @@ function similar(last: ScrollState, current: ScrollState): boolean {
     return (dx * dx + dy * dy < Setting.Distance * Setting.Distance) && (current.time - last.time < Setting.Interval);
 }
 
+export function compute(): void {
+    if (initTopNode || initBottomNode) {
```

Comment:
>initTopNode || initBottomNode [](http://example.com/codeflow?start=8&length=29)

nit but if you are going to have this outer check you should split it into two separate if clauses (one for top and one for bottom)